### ISSUE TITLE
Data structure

### ISSUE THREAD

**Issue by fly2mun** [enoche/IGT-WSDM23#1]
Thanks for your code.

Please tell me what the meaning of s0~s56, a0~a120, and r0~r84 in the toy_process data.

Thanks.
Moon.

**Comment by enoche** [comment]
Hi, @fly2mun 
Thank you for your interest in our paper. Here's a brief overview of the features we've used in toy dataset:

1. **Sender (Retailer) Feature**: This feature has 57 dimensions, providing a comprehensive profile of the sender.

2. **Order Accepted Feature (Origin City)**: This feature, with its 121 dimensions, gives detailed information about the city where the order was accepted.

3. **Receiver Feature**: This feature has 85 dimensions and offers an in-depth understanding of the receiver.

We hope this helps you understand our work better. If you have any more questions, feel free to ask.

**Comment by fly2mun** [comment]
Thank you  for replying email.
I am trying to learn with toy data, but the learning continues to repeat
infinitely.

I would like to create the data myself.
Can you tell me what dimention means detail.

I can't do anything with numbers now.

Please tell me in more detail?

Thanks, Moon

2023년 10월 24일 (화) 오전 11:18, Zhou Xin ***@***.***>님이 작성:

**Comment by enoche** [comment]
Hello, thanks for your feedback.
The purpose of `toy` dataset is for running up of the code. As the `toy` data is generated with `https://github.com/enoche/IGT-WSDM23/blob/master/datasets/toy_gen.ipynb`, hence not ensure the algorithm can be converged on this dataset.

You are encouraged to check `https://github.com/enoche/IGT-WSDM23/blob/master/datasets/toy_gen.ipynb` for generating more data.
Please reply in case we could further assist your research.

**Comment by fly2mun** [comment]
Thanks you for your kind reply.

What I want to know is roughly what the sendor feature is. It seems to be the seller's ID or other information.
And oring city seems to contain GPS data and other contents.

In the case of receiver features, I think it refers to distance, weather, etc.

I am curious about what information went into creating the feature.

Can i know the items of features list?

Thanks
Moon.

**Comment by enoche** [comment]
Indeed, I understand your concerns. However, I regret to inform you that the detailed information regarding how the generated data correlates with the meaning of each feature is beyond my comprehension. As far as I know, the features encompass statistical information derived from previous orders.

And you are encouraged to test the model on your own data/features~  Thanks.

**Comment by saihow1999** [comment]
Hi @enoche,
I have also one doubt on the above mentioned. The `toy_gen.ipynb` does not generate the embeddings. Point out if I am wrng. I am very eager to know that how you have generated those embeddings. I can unserstand you cant share the exact code but it would be really helpful if you can throw some light on it.
